templates/sample-app/res/src/apex/force-app/main/default/classes/HerokuConnector.cls

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) 2017-2018, FinancialForce.com, inc
```

Comment:
Looks all 2018-ish to me

Reply:
Some of these files have been moved so for this stuff I'm happier to leave it as 2017-18.